pydocx/export/base.py

```diff
@@ -547,7 +573,5 @@ def export_textbox_content(self, textbox_content):
         return self.yield_nested(textbox_content.children, self.export_node)
 
     def export_markup_compatibility_alternate_content(self, alternate_content):
-        return self.yield_nested(alternate_content.children, self.export_node)
-
-    def export_markup_compatibility_fallback(self, fallback):
-        return self.yield_nested(fallback.children, self.export_node)
+        if self.first_pass:
+            self.alternate_contents.append(alternate_content)
```

Comment:
I don't quite understand why alternate contents need to be captured. We had to capture complex fields because they would span across runs. That doesn't seem to be the case with ACs. I was expecting you would have a method here like `self._unwrap_alternate_content_fallback` and you would pass in the alternate content.

Then, that method would update the parent of each child in the fallback to match the AC's parent, and add them as children of the AC's parent.